plot_data/core.py

```diff
@@ -790,7 +796,7 @@ def __init__(self, graphs: List[Dataset], x_variable: str, y_variable: str,
             self.axis = axis
         self.log_scale_x = log_scale_x
         self.log_scale_y = log_scale_y
-        PlotDataObject.__init__(self, type_='graph2d', name=name)
+        Figure.__init__(self, width=width, height=height, type_='graph2d', name=name)
```

Comment:
Can use super().__init__() in order to avoid further maintenance. In addition to that, it enables to omit the "self" argument